src/components/navbar/downloadPanel/DownloadPanel.tsx

```diff
@@ -150,6 +157,18 @@ function DownloadPanel(props: DownloadPanelProps) {
                     )}
                 </DownloadPanelContainer>
             </Menu>
+            <ErrorToastWrapper>
```

Comment:
can it be somewhere on the top level for further re-use? And yes we would need introduce new context to handle it from different places of the components. Another way to propagate the error to Platfrom level via callback